src/forms/button/Button.tsx

```diff
@@ -44,10 +45,14 @@ export enum ButtonSize {
 // TODO: add loading support
 
 export class Button extends React.PureComponent<ButtonProps> {
+    static defaultProps = {
+        isPrimaryClassRequired: true,
+    };
+
     private static getClassNames(props: ButtonProps): (string | undefined)[] {
         return [
-            "btn",
-            props.icon ? "btn-icon" : undefined,
+            props.isPrimaryClassRequired ? "btn" : undefined,
+            props.isPrimaryClassRequired ? (props.icon ? "btn-icon" : undefined) : undefined,
```

Comment:
Here I want to make "btn" and "btn-icon" class as configurable. PFA SS for the reference for with and without class effect.
<img width="784" alt="Header Button without btn and btn-icon class" src="https://user-images.githubusercontent.com/12778040/61269722-19803180-a7bd-11e9-960e-497a8cd589c9.png">
<img width="576" alt="Button with and without classess" src="https://user-images.githubusercontent.com/12778040/61269723-1a18c800-a7bd-11e9-8d9f-b6630190e169.png">
<img width="784" alt="Header Button with btn and btn-icon class" src="https://user-images.githubusercontent.com/12778040/61269724-1a18c800-a7bd-11e9-8df0-c83886556a75.png">

Reply:
@vinodwani18 I got your problem. You can try something as mention below.
Name your prop as say 'customBtn' . By default it will be false. 
If customBtn is true (i.e in your case you need to pass it as true) then it will not apply 'btn' and 'btn-icon' class.

Reply:
Done